mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping

```diff
@@ -0,0 +1,80 @@
+CLASS net/minecraft/class_5455 net/minecraft/util/registry/DynamicRegistryManager
+	COMMENT A manager of dynamic registries. It allows users to access non-hardcoded
+	COMMENT registries reliably.
+	COMMENT
+	COMMENT <p>Each minecraft server has a dynamic registry manager for file-loaded
+	COMMENT registries, while each client play network handler has a dynamic registry
+	COMMENT manager for server-sent dynamic registries.</p>
+	COMMENT
+	COMMENT <p>The {@link DynamicRegistryManager.Impl}
+	COMMENT class serves as an immutable implementation of any particular collection
+	COMMENT or configuration of dynamic registries.</p>
+	FIELD field_25918 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_25919 INFOS Ljava/util/Map;
+	METHOD method_30518 getDimensionTypes ()Lnet/minecraft/class_2378;
+	METHOD method_30519 load (Lnet/minecraft/class_3300;)Lnet/minecraft/class_5455$class_5457;
+		COMMENT Loads a dynamic registry manager from the resource manager's data files.
+		ARG 0 resourceManager
+	METHOD method_30520 register (Lcom/google/common/collect/ImmutableMap$Builder;Lnet/minecraft/class_5321;Lcom/mojang/serialization/MapCodec;Z)Lcom/google/common/collect/ImmutableMap$Builder;
+		COMMENT Registers a registry handling logic to the immutable map builder for
+		COMMENT registry infos.
+		ARG 0 infosBuilder
+		ARG 1 registryRef
+		ARG 2 elementCodec
+		ARG 3 synced
+	METHOD method_30524 addBuiltinEntries (Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_2378;)V
+		COMMENT Add all entries of the {@code registry} to the corresponding registry
```

Comment:
```suggestion
		COMMENT Add all built-in entries of the {@code registry} to the corresponding registry
```

Reply:
Nah it just adds all the entries in the argument registry to the argument `DynamicRegistryManager.Impl`. The fact the argument may contain all the default values is a coincidence.

Reply:
Oops, yes, this was on the wrong method, it was supposed to be a suggestion for the method that takes the key of a registry and pulls all entries from BuiltinRegistries.